screens/Content/ContentScreen.jsx

```diff
@@ -1,12 +1,12 @@
-import {useContext} from 'react';
+import React, {useContext} from 'react';
```

Comment:
why you need this?

Reply:
Two days ago I removed the React import, but after installing ESLint I got a warning like this : 
"ESLint: 'React' must be in scope when using JSX (react/react-in-jsx-scope) " . 

After importing React, the warning disappears.

Reply:
Which React version do you use? If it's one of the last versions, then you should update(specify React version) eslint

Reply:
I checked the versions of ESLint (8.41.0) and React JS (18.2.0) and at the moment these versions are the latest for both . Advise, please, does it make sense to update something?